src/libraries/System.Security.Permissions/src/System/Security/SecurityManager.cs

```diff
@@ -6,6 +6,9 @@
 
 namespace System.Security
 {
+#if NET50_OBSOLETIONS
```

Comment:
Oops -- I missed the ref assembly for this round of changes to CAS.  Addressing...